app/src/redux/contactInfo.js

```diff
@@ -0,0 +1,50 @@
+import { messages, request, API } from '../helper';
+
+const SHOW_MODAL = 'SHOW_MODAL';
+const CONTACT_REQUEST = 'CONTACT_REQUEST';
+const MESSAGE_MODAL = 'MESSAGE_MODAL';
+
+export default function changeReducer(state = {show: false, message: ''}, action) {
+  switch (action.type) {
+    case SHOW_MODAL:
+      return Object.assign({}, state, {show: action.show});
+    case MESSAGE_MODAL:
+      return Object.assign({}, state, {message: action.message});
+    default:
+      return state;
+  }
+}
+
+export function showModal(show) {
+  return {
+    type: SHOW_MODAL,
+    show: show
+  };
+}
+
+export function messageModal(message) {
+  return {
+    type: MESSAGE_MODAL,
+    message: message
+  };
+}
+
+export function contactRequest() {
+  return {type: CONTACT_REQUEST};
+}
+
+export function contactInfo({name, surname, email, textarea}) {
```

Comment:
" this.props.actions.contactInfo(this.state) " this is from your component app/src/containers/Contacts/Contacts.js. So why don't not just `export function contactInfo(data)` ?